packages/generator-rest/test/poc/promotion-data.ts

```diff
@@ -0,0 +1,80 @@
+/* !
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import {
```

Comment:
Is this data used for something?

Reply:
This is a json object (80 lines) used as the request body of the e2e test.